pandas/sparse/tests/test_frame.py

```diff
@@ -879,7 +892,15 @@ def test_numpy_cumsum(self):
         tm.assertRaisesRegexp(ValueError, msg, np.cumsum,
                               self.frame, out=result)
 
+    def test_numpy_func_call(self):
+        # no exception should be raised even though
+        # numpy passes in 'axis=None'
+        funcs = ['sum', 'cumsum', 'var', 'mean', 'prod',
+                 'cumprod', 'std']
```

Comment:
`np.argmin` or `np.argmax` breaks because `__array_wrap__` expects a 2-D input, which is not what you get when you call either of those functions.  I think this is a bug on the `pandas` side because `argmin` and `argmax` should return scalars, meaning `__array_wrap__` should have to accept them.  @jreback thoughts?